### PR TITLE
Add back Consumer.Offsets.CommitInterval default value #1616

### PR DESCRIPTION
**Pull request by reenjii**
https://github.com/Shopify/sarama/issues/1616

### CODE REVIEW

**Comment by reenjii** [comment]
I signed Shopify's CLA, please re-launch the check. Thanks.

File: config.go [hunk 1]
```diff
@@ -457,6 +457,7 @@ func NewConfig() *Config {
 	c.Consumer.Return.Errors = false
 	c.Consumer.Offsets.AutoCommit.Enable = true
 	c.Consumer.Offsets.AutoCommit.Interval = 1 * time.Second
+	c.Consumer.Offsets.CommitInterval = 1 * time.Second
```
Comment (d1egoaz) [inline config.go]: @reenjii I'm wondering if we need to add back the initial value....
We added this here: https://github.com/Shopify/sarama/pull/1590
image

But if we add back the initial value that would means that by default sarama will *always* log a deprecated message.

Couldn't this be solved by changing the value when creating the sarama-cluster config? as the field was added back

Comment (reenjii) [reply config.go]: Yeah I understand that it would look wierd.
I'll add this default value in our configuration then.

**Comment by d1egoaz** [comment]
closed because of https://github.com/Shopify/sarama/pull/1618#discussion_r380694740

Thanks anyways @reenjii

**Pull request closed without merging**